src/ramses_tx/ramses.py

```diff
@@ -273,6 +273,8 @@
     Code._1298: {  # co2_level
         SZ_NAME: "co2_level",
         I_: r"^00[0-9A-F]{4}$",
+        RQ: r"^00$",  # tried in ramses CLI, got reply:
+        RP: r"^00[0-9A-F]{4}$",
```

Comment:
When the `RP` payload REGEX is the same as the `I`, then you don't add the RP, but you do need to have the `RQ`.

That is what this code (later in **ramses.py** does:
```python
for code in CODES_SCHEMA.values():  # map any RPs to (missing) I_s
    if RQ in code and RP not in code and I_ in code:
        code[RP] = code[I_]
```

So, please remove the `RP: r"^00[0-9A-F]{4}$",`